examples/tutorial/index.js

```diff
@@ -0,0 +1,306 @@
+'use strict';
+
+const cheerio = require('cheerio');
+const { dom, out, props, rule, ruleset, score, type } = require('fathom-web');
+const Futils = require('fathom-web/utils');
+const JSDOM = require('jsdom/lib/old-api');
+const Libingester = require('libingester');
+const url = require('url');
+const util = require('util');
+const { Vimeo } = require('vimeo');
+const Youtubedl = require('youtube-dl');
+
+Youtubedl.getInfo = util.promisify(Youtubedl.getInfo);
+
+const feedURI = 'https://creativecommons.org/blog/feed/';
```

Comment:
There is no need for a `const` here.

Reply:
The original tutorial has it. I think is good to have the feed in a const.

Reply:
Well, you declare a `const` only to use it in one place.

Reply:
I understand declaring a var or const is useful when you reuse it. But is also good for readability. Honestly I prefer the feed URL at the top in a const, especially in a tutorial.